### PR TITLE
[Bug Fix] Index.d.ts not being generated for @bottender/dialogflow

### PR DESCRIPTION
**Pull request by preetjdp**
**What does this do?**

Fixes a bug wherein `index.d.ts` was not being generated properly.

**Notes**

I have not bumped the version and/or updated the `CHANGELOG` file.
If that is required from my end. Please do let me know.

### CODE REVIEW

File: packages/bottender-dialogflow/src/index.ts [hunk 1]
```diff
@@ -30,7 +30,7 @@ function getFulfillments(fulfillmentMessages: Message[]): string[] {
  *   },
  * });
  */
-module.exports = function dialogflow({
+export default function dialogflow({
```
Comment (chentsulin) [inline packages/bottender-dialogflow/src/index.ts:33]: This should be `export = function dialogflow({` in TS

Using `export default` here will introduce a breading change to commonjs users:

```js
const dialogflow = require('@bottender/dialogflow').default;
```

Comment (preetjdp) [reply packages/bottender-dialogflow/src/index.ts:33]: Hi doing `export = function dialogflow` did not work.

Did you mean
`export function dialogflow({`

Comment (chentsulin) [reply packages/bottender-dialogflow/src/index.ts:33]: What does `did not work` mean?

Actually, we have done this in release/1.5 branch.

https://github.com/Yoctol/bottender/blob/v1.5.1-alpha.5/packages/bottender-dialogflow/src/index.ts#L56

Comment (preetjdp) [reply packages/bottender-dialogflow/src/index.ts:33]: Oh, Superb.
Did not know that.

Will close this PR.

**Pull request closed without merging**